python/mlcroissant/mlcroissant/_src/core/optional.py

```diff
@@ -86,5 +86,10 @@ def PIL_Image(cls) -> types.ModuleType:  # pylint: disable=invalid-name
         """Cached git module."""
         return _try_import("PIL.Image", package_name="Pillow")
 
+    @cached_class_property
+    def LIB_Audio(cls) -> types.ModuleType:  # pylint: disable=invalid-name
```

Comment:
`def librosa` (PIL.Image above is a library, so the name here should probably be librosa)

Reply:
I just changed the name, and the usage in field.py